mmdeploy/codebase/mmrotate/deploy/rotated_detection.py

```diff
@@ -13,6 +14,28 @@
 from .mmrotate import MMROTATE_TASK
 
 
+def replace_RResize(pipelines):
```

Comment:
What's the motivation for `replace_RResize`? Will it affect model converter?

Reply:
for testing, there is no difference between them.